ansible/cleanup/roles/version/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
yes I had realized that later and removed it in a commit later https://github.com/arcolife/cfme-performance/commit/6f5f938397b1501339f644779559c1d9c8b93f40